grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -693,6 +693,16 @@ class StockMovementService {
                             newItem.boxName = requisitionItem.boxName
                             newItem.lotNumber = requisitionItem.lotNumber
                             newItem.expirationDate = requisitionItem.expirationDate
+                            newItem.requisition = requisition
```

Comment:
NOTE: This happens automatically in line 706, but it's fine to do this explicitly so feel free to leave it.